openhtf/io/output/mfg_inspector.py

```diff
@@ -381,11 +382,14 @@ def locked_get(self):
     def locked_put(self, credentials):
       self._credentials = credentials
   # pylint: enable=invalid-name,missing-docstring
-
-  def __init__(self, user, keydata, token_uri=TOKEN_URI):
+  
+  def __init__(self, user, keydata, token_uri=TOKEN_URI,
+               error_message=None, filename_pattern_fail=None):
```

Comment:
The idea is being able to trigger an alert for the operators that something is wrong.  Would give the ability to add yellow/bold warning text.  I should add a default message.